README.md

```diff
@@ -48,6 +48,10 @@ git fetch origin
 git reset --hard origin/master
 ```
 
+### How do I run `brew update`?
+
+To be able to clone the repository into `/usr/local`, run `git config --global --add safe.directory /usr/local`.
```

Comment:
Thank you! Maybe worth specifying that users should do this if they have trouble running `brew update`?
```suggestion
If you have trouble running `brew update` when your installation is located in `/usr/local`, run `git config --global --add safe.directory /usr/local`.
```